label_studio/data_manager/prepare_params.py

```diff
@@ -41,3 +42,249 @@ class PrepareParams(BaseModel):
+        'You can use [the helper class `Filters` from this page](https://labelstud.io/sdk/data_manager.html) '
+        'to create Data Manager Filters.<br>'
+        'Example: `{"conjunction": "or", "items": [{"filter": "filter:tasks:completed_at", "operator": "greater", '
+        '"type": "Datetime", "value": "2021-01-01T00:00:00.000Z"}]}`'
+    )
+)
+
+selected_items_schema = openapi.Schema(
+    type=openapi.TYPE_OBJECT,
+    required=['all'],
+    description='Task selection by IDs. If filters are applied, the selection will be applied to the filtered tasks.'
+                'If "all" is `false`, `"included"` must be used. If "all" is `true`, `"excluded"` must be used.<br>'
+                'Examples: `{"all": false, "included": [1, 2, 3]}` or `{"all": true, "excluded": [4, 5]}`',
+    oneOf=[
+        openapi.Schema(
+            title='all: false',
+            type=openapi.TYPE_OBJECT,
+            properties={
+                'all': openapi.Schema(type=openapi.TYPE_BOOLEAN, enum=[False], description='No tasks are selected'),
+                'included': openapi.Schema(
+                    type=openapi.TYPE_ARRAY,
+                    items=openapi.Schema(type=openapi.TYPE_INTEGER),
+                    description='List of included task IDs'
+                )
+            },
+            required=['all']
+        ),
+        openapi.Schema(
+            title='all: true',
+            type=openapi.TYPE_OBJECT,
+            properties={
+                'all': openapi.Schema(type=openapi.TYPE_BOOLEAN, enum=[True], description='All tasks are selected'),
+                'excluded': openapi.Schema(
+                    type=openapi.TYPE_ARRAY,
+                    items=openapi.Schema(type=openapi.TYPE_INTEGER),
+                    description='List of excluded task IDs'
+                )
+            },
+            required=['all']
+        )
+    ]
+)
+
+# Define ordering schema
+ordering_schema = openapi.Schema(
+    type=openapi.TYPE_ARRAY,
+    items=openapi.Schema(
+        type=openapi.TYPE_STRING,
+        enum=Column.enums_for_ordering(),
+    ),
+    description='List of fields to order by. Fields are similar to filters but without the `filter:` prefix. '
+                'To reverse the order, add a minus sign before the field name, e.g. `-tasks:created_at`.'
+)
+
+# Define the main schema for the data payload
+data_schema = openapi.Schema(
+    type=openapi.TYPE_OBJECT,
+    properties={
+        'filters': filters_schema,
+        'selectedItems': selected_items_schema,
+        'ordering': ordering_schema
+    },
+    description='Additional query to filter and order tasks'
+)
+
+prepare_params_schema = openapi.Schema(
+    type=openapi.TYPE_OBJECT,
+    properties={
+        'filters': filters_schema,
+        'selectedItems': selected_items_schema,
+        'ordering': ordering_schema
+    },
+    description='Data payload containing task filters, selected task items, and ordering'
```

Comment:
Can we add examples here? similar to https://github.com/HumanSignal/label-studio/blob/develop/label_studio/tasks/openapi_schema.py#L70
Then they would become visible under `-d` here:
<img width="542" alt="image" src="https://github.com/HumanSignal/label-studio/assets/6087484/008059aa-8348-4c05-9c9a-330d1f37c277">